CITATION.cff

```diff
@@ -0,0 +1,56 @@
+cff-version: 1.2.0
```

Comment:
This file comes from the template, but we have realized that if it is included, then pipeline releases on Zenodo only reference the nf-core publication. So it's better to remove it

Reply:
Do you mean removing the whole file?

Reply:
yes exactly